docs/API.Auth.InvalidateCache.md

```diff
@@ -0,0 +1,32 @@
+# Function: *Invalidate Cache* #
+
+## Description ##
+
+CM-Well manages a cache with authentication information for better performance. If you've made a change to authentication information such as creating a new user, creating a new role or changing a password, and you want the change to take immediate effect, you will need to refresh this cache. You do this by calling the **invalidate-cache** API.
+
+>**Notes:** 
+>* You will need an access token with admin permissions to call this API.
+>* If you have several CM-Well nodes, then in order to refresh the caches on all nodes, you'll need to send a separate **invalidate-cache** command to each node in your cluster.
+
+## Syntax ##
+
+**URL:** \<CMWellHost\>/_auth?op=invalidate-cache -H "X-CM-Well-Token:\<AdminToken\>"
+**REST verb:** GET
+**Mandatory parameters:** Admin-permissions token in the **X-CM-Well-Token** header
+
+## Code Example ##
+
+### Call ###
+
+    <CMWellHost>/_auth?op=invalidate-cache -H "X-CM-Well-Token:<AdminToken>"
+
+### Results ###
+
+    {"sucess":true}
```

Comment:
as you wrote on another place - I think it is best if also mentioned here - on rare cases response might be "success: false". In that case, just try again. (this is because the cache might be just updating itself in the exact time of the invalidation request, so it is "locked" and will report the operation as unsuccessful).